Frameworks/UIKit/UITouch.mm

```diff
@@ -225,7 +226,14 @@ - (CGPoint)locationInView:(UIView*)viewAddr {
     if (viewAddr != 0) {
         UIWindow* window = [viewAddr window];
         if (window != nil) {
-            ret = [window _convertPoint:location fromView:window toView:viewAddr];
+            UIWindow* fromWindow = window;
+
+            if (GetCACompositor()->IsRunningAsFramework()) {
+                // In framework mode the parent UIWindow is not visible
+                fromWindow = nil;
+            }
+
+            ret = [window _convertPoint:location fromView:fromWindow toView:viewAddr];
```

Comment:
UIView's point conversion method doesn't really work here because there's no UIView that we're converting _from_.

---
In reply to: [113980192](https://github.com/Microsoft/WinObjC/pull/2650#discussion_r113980192) [](ancestors = 113980192,113979397)